src/main/java/emissary/util/PayloadUtil.java

```diff
@@ -67,16 +96,60 @@ public static String getPayloadDisplayString(final IBaseDataObject payload) {
         sb.append("\n").append("filename: ").append(fileName).append("\n").append("   creationTimestamp: ").append(creationTimestamp).append("\n")
                 .append("   currentForms: ").append(currentForms).append("\n").append("   filetype: ").append(payload.getFileType()).append("\n")
                 .append("   transform history (").append(th.size()).append(") :").append("\n");
-        for (final TransformHistory.History h : th) {
-            sb.append("     ");
-            if (h.wasCoordinated()) {
-                sb.append("  ");
-            }
-            sb.append(h.getKey()).append("\n");
+
+        // transform history output
+        switch (configureHistoryCase(fileName)) {
+            case REDUCED_HISTORY:
+                // found reduced history match, output only dropoff
+                sb.append("   ** reduced transform history **").append("\n").append("     dropOff -> ")
+                        .append(payload.getLastPlaceVisited())
+                        .append("\n");
+                break;
+            case NO_URL:
```

Comment:
Cases `NO_URL` and `default` are almost identical, can we squash these together?

Reply:
I could, then just move the check for the NO_URL inside the default to alter the h.getKey() or h.getKeyNoUrl(). Whichever way of checking that you think is better

Reply:
Squashed NO_URL and default, left as switch statement for now for possible future transform history altercations.